packages/cli/src/utils.ts

```diff
@@ -342,15 +342,16 @@ export function parseCommaSeparatedList(value: string): string[] {
 export function writeTSConfig(basePath: string, fileList: string[]): void {
   const include = [...fileList.map((f) => f.replace('.js', '.ts'))];
   const config = {
-    $schema: 'http://json.schemastore.org/tsconfig',
+    $schema: 'https://json.schemastore.org/tsconfig',
     compilerOptions: {
-      baseUrl: '.',
-      outDir: 'dist',
-      emitDeclarationOnly: true,
-      allowJs: true,
-      target: 'es2016',
-      module: 'commonjs',
+      strict: true,
+      noUncheckedIndexedAccess: true,
+      module: 'es2020',
       moduleResolution: 'node',
+      newLine: 'lf',
+      target: 'ES2021',
+      forceConsistentCasingInFileNames: true,
+      noFallthroughCasesInSwitch: true,
```

Comment:
Don't we want `"noEmit": true` to be in here since we are removing the `outDir`?

Reply:
@chadhietala yes. depending on type inference work we might need to generate the d.ts files. but yeah as far as i can see we do not want the compiler output files.

Reply:
👍